src/yunohost/backup.py

```diff
@@ -1543,23 +1544,37 @@ def _organize_files(self):
             if not os.path.isdir(dest_dir):
                 filesystem.mkdir(dest_dir, parents=True)
 
-            # Try to bind files
+            # For directory, attempt to mount bind
             if os.path.isdir(src):
                 filesystem.mkdir(dest, parents=True, force=True)
-                ret = subprocess.call(["mount", "-r", "--rbind", src, dest])
-                if ret == 0:
-                    continue
+
+                mount_commands = []
+                mount_commands.append("mount --rbind %s %s" % (src, dest))
+                mount_commands.append("mount -o remount,ro,bind %s" % dest)
+
+                try:
+                    run_commands(mount_commands)
```

Comment:
I think subprocess.call is better, how runcommand protects from a dest arg with malicious code ?